### PR TITLE
Update protonmail.toml

### PR DESCRIPTION
**Pull request by erotavlasme**
**Type of pull request:** product edit

**Related issues:** changed rubric.security, added independent audit
---

### CODE REVIEW

File: products/protonmail.toml [hunk 1]
```diff
@@ -51,9 +51,9 @@ value = "yes"
...
 
 [rubric.security]
-value = "yes"
+value = "yes-independent-audits"
 citations = ["All servers used in connection with the provisioning of the Service are located in Switzerland and wholly owned and operated by the Company. Only employees of the Company have physical or other access to the servers. Data is ALWAYS stored in encrypted format on our servers. Offline backups may be stored periodically, but these are also encrypted. We do not possess the ability to access any user encrypted message content on either the production servers or in the backups."]
-notes = ["ProtonMail has once received an independent audit, though these are not mentioned in the policy as their official security measure."]
+notes = ["[All ProtonMail apps are now open source and audited. [openPGP library](https://protonmail.com/blog/openpgpjs-protonmail-security-audit/) [Android](https://protonmail.com/blog/android-open-source/), [iOS](https://protonmail.com/blog/ios-open-source/), [bridge](https://protonmail.com/blog/bridge-open-source/)."]
```
Comment (doamatto) [inline products/protonmail.toml:56]: Maybe rephrase to say `All ProtonMail apps are now open source and audited. See: [...]`

**Review by doamatto — CHANGES_REQUESTED** [review]
One tiny change I would make; but apart from that, looks solid to me

File: products/protonmail.toml (hunk 1 above)
Comment (erotavlasme) [reply products/protonmail.toml:56]: Ok

**Review by doamatto — APPROVED** [review]

**Comment by milesmcc** [comment]
Nice. Thanks @erotavlasme and @doamatto!
